wiki/osu!_Program_Files/en.md

```diff
@@ -1,113 +1,117 @@
----
-needs_cleanup: true  # needs to be rewritten (issue #4753)
----
-
 # osu! program files
 
-![osu! Program Files in a nutshell](img/osu!-program-files.png "osu! Program Files in a nutshell")
+![The file structure of osu!'s installation folder, on Windows and macOS](img/file_structure.jpg "The file structure of osu!'s installation folder, on Windows and macOS")
 
-## Location
+The **osu! program files** are a set of files that run osu! and keep track of different activities as user's play.
 
-### Windows
+## Installation paths
 
-| Current default filepath | Old installer filepath |
-| :-- | :-- |
-| `C:\Users\<Username>\AppData\Local\osu!` | `C:\Program Files\osu!` / `C:\Program Files(x86)\osu!` (for 7 64-bit or 8) |
+osu! is installed in the following locations, by default:
 
-> `C:` is your drive with osu! installed.
+| Windows | macOS |
+| :-- | :-- |
+| `C:\Users\<Username>\AppData\Local\osu!` | `/Applications/osu!.app/Contents/drive_c/osu!` |
 
 ## Folders
 
 ### Chat
 
-Only appears when you used "/savelog" at Chat Console once or if you enabled "Automatically log private messages" in Options. The folder shows your **Chat Console tab's log of comments.**
+The Chat folder only appears if the player has "Automatically log private messages" enabled in Options, or the player runs the "/savelog" command in the [Chat Console](/wiki/Chat_Console).
 
-The styling is `{Tab_name}-{YYYYMMDD}-{HHMMSS}` in .txt format, which basically can be opened in any word application of choice(Notepad).
+The name structure of the files is `{Tab_name}-{YYYYMMDD}-{HHMMSS}`, and can be opened in any text editor. An example of this is shown below:
 
-**Example:** #multiplayer-20121115-040845 (/savelog at #multiplayer tab in 15th November 2012 at 04hrs, 08mins, and 45secs)).
+**Example**: #multiplayer-20121115-040845 (/savelog at #multiplayer tab in 15th November 2012 at 04hrs, 08mins, and 45secs).
```

Comment:
if you have an "an example is shown below" up top, you dont need to have this here. this line should also be surround by grave marks as well

Reply:
Fixed in 49d39e3